apps/backend/src/modules/auth/auth.ts

```diff
@@ -1,10 +1,17 @@
+import { config } from "~/libs/modules/config/config.js";
 import { logger } from "~/libs/modules/logger/logger.js";
 import { userService } from "~/modules/users/users.js";
 
+import { TokenService } from "../../libs/modules/token/token.service.js";
```

Comment:
let's rename it to `token`, and it should be created the same way as the logger or config module. We should re-export a single instance from this module 
```suggestion
import { token } from "~/libs/modules/token/token.js";
```

Reply:
Should I create also a libs/types folder, and declare types for this service? If so, should I name that type as a Token, or a TokenService?

Reply:
Fixed. Also added types for this module, as in the logger.